test-harness/build-base-image.sh

```diff
@@ -23,6 +24,11 @@ function echoIfVerbose() {
         echo "${@}"
     fi
 }
+
+if [ -f ~/.bash_profile ]; then
+  . ~/.bash_profile
```

Comment:
Should we be relying on the user to have executed this in their shell prior to invoking the script? I believe (correct me if I'm wrong here) that if they do this, the environment set in `.bash_profile` is inherited by the child process running the scripts.

Reply:
I've altered how this operates. Instead of loading the bash profile, the gnu-getopt installation location is loaded into a variable only within the scope of the script's execution.